pysam/libchtslib.pyx

```diff
@@ -222,6 +226,8 @@ cdef class HTSFile(object):
                 fd = self.filename
             else:
                 fd = self.filename.fileno()
+                
+            dup_fd = dup(fd)
```

Comment:
Yes, we can. Probably close_file = False and dup should be default? 
Otherwise I can see no way to prevent double closing errors when used with File objects.